filebrowser_safe/storage.py

```diff
@@ -137,10 +138,10 @@ def isdir(self, name):
         if self.isfile(name):
             return False
 
-        name = self._normalize_name(self._clean_name(name))
-        dirlist = self.listdir(self._encode_name(name))
+        (dirs, files) = self.listdir(name)
+        dirlist = itertools.chain(dirs, files)
 
-        # Check whether the iterator is empty
+        # Check for contents
         for item in dirlist:
```

Comment:
the `dirlist` returned by `listdir()` is a tuple of `(dirs, files)`, so the iteration check was previously not testing anything about the listing -- (it would always return `True`).